src/Components/About/About.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import Footer from '../Footer/Footer.jsx'
+import SignUp from '../MainPage/SignUp/SignUp.jsx'
+
+
+
+function About() {
```

Comment:
Please use Class declaration to use react benefits.
e.g. `export default class About extends React.Component...`